### ISSUE TITLE
readme documentation for quickcheck_macros is outdated

### ISSUE THREAD

**Issue by Charles-Johnson** [BurntSushi/quickcheck#227]
I am trying to use quickcheck in my integrations tests. At the beginning one of the test files I have added:
```rust
#![feature(plugin)]
#![plugin(quickcheck_macros)]

extern crate my_project;
#[cfg(test)]
extern crate quickcheck;
```
followed by `use` statements and `#[test]` functions definitions.

In my `Cargo.toml` file I have:
```toml
[dev-dependencies]
quickcheck = "0.8"
quickcheck_macros = "0.8"
```
When compiling `my_project` I get this
```
error: /home/charles/my_workspace/target/debug/deps/libquickcheck_macros-4d9dd5db3e15aaa1.so: undefined symbol: __rustc_plugin_registrar_e32604afeb7083bcd5f34dda9dbf0cd__
  --> my_project/tests/definitions.rs:19:11
   |
19 | #![plugin(quickcheck_macros)]
   |           ^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `my_project`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by BurntSushi** [comment]
The README looks out dated. You shouldn't need `#![feature(plugin)]` or `#![plugin(quickcheck_macros)]` any more. It now works on Rust stable. See: https://github.com/BurntSushi/quickcheck/blob/master/quickcheck_macros/examples/attribute.rs

**Comment by BurntSushi** [comment]
It looks like this was fixed in 8ec8cb1dc9db4cfbdf5fe6898377168b0c4a26e6.
